operator/endpointupdater/docker/docker.go

```diff
@@ -0,0 +1,5 @@
+package docker
+
+type Docker struct {
+	Image string
```

Comment:
I realized that certain structures in the clustertpr still miss JSON and YAML tags. Will fix this in the next PR separately.